### PR TITLE
Data Modal PR

### PR DESCRIPTION
**Pull request by Javed-Shahbaz**
- Create a Blog-App
- Initialize the project with Git.
- Postgres database set up.
- Build project schema.
- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
- Foreign keys included.

### CODE REVIEW

**Comment by MahmoudBakr23** [comment]
### Hi @Javed-Shahbaz 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

**Invalid Submission**
- Please add a professional README file using the MIcroverse template

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Comment by binhussen** [comment]
@Javed-Shahbaz would you fix the broken `mit` link in your readme

File: README.md [hunk 1]
```diff
@@ -1 +1,177 @@
...
+I would like to thank to everyone who watch my code and like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License.md) licensed.
+
```
Comment (binhussen) [inline README.md]: - Kindly, the `mit` link is broken, please fix it to have a professional readme 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Javed-Shahbaz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Created a data model ✔️
- Well Structured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Javed-Shahbaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes from the previous review (✅)) 👏
- Created migration files in order to set up database tables👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**